build-tools/installers/sign-pkg-content.proj

```diff
@@ -43,8 +43,24 @@ ourself (using an empty signing identity) before passing these files to ESRP.
     <RemoveDir Directories="$(_BundleToolExtractLocation)" />
   </Target>
 
+  <!-- Remove the jna subdirectory in manifestmerger.jar as it should not be needed by our usage.
+    This simplifies signing/notarization and reduces our installation footprint slightly. -->
+  <Target Name="_RemoveJnaFromManifestMerger" >
```

Comment:
Maybe instead we should update:

https://github.com/xamarin/xamarin-android/blob/3e27d4b4c647090bcb5994d77e7de6ada2123879/src/manifestmerger/build.gradle#L41

To do something like: `'com/sun/jna/*/*jnidispatch*'`

Reply:
Thanks, this is a much better approach.